service/build.gradle

```diff
@@ -74,6 +74,9 @@ dependencies {
     testImplementation('au.com.dius.pact.provider:junit5spring:4.3.19')
     testImplementation('au.com.dius.pact.consumer:junit5:4.3.19')
     testImplementation("au.com.dius.pact.consumer:junit:4.3.19")
+    testImplementation('org.testcontainers:junit-jupiter')
+    testImplementation('org.testcontainers:postgresql')
+    testImplementation('org.springframework.boot:spring-boot-testcontainers:3.2.3')
```

Comment:
Could we extract testcontainers into a separate PR. It'd be nice to remove the noise from the clone recovery changes AND make the PRs more targeted and easier to find if we want to reference them in the future